qdrant-landing/content/documentation/cloud-account-setup.md

```diff
@@ -23,7 +25,7 @@ Every account is tied to an email address. You can invite additional users to yo
 
 ## Inviting additional users to an account
 
-You can invite additional users to your account, and manage their permissions on the *Account Management* page in the Qdrant Cloud Console.
+You can invite additional users to your account, and manage their permissions on the *Account -> Access Management* page in the Qdrant Cloud Console.
```

Comment:
`Account -> Access Management` part is not highlighted properly. It is not much distinguishable from other words. Maybe we should use inline code or bold text for better highlighting.
![image](https://github.com/user-attachments/assets/2321607e-424b-4733-8673-269d81332c57)